components/arm.py

```diff
@@ -0,0 +1,29 @@
+import wpilib
+
+
+class Arm:
+    """
+    Bunny grabber arm.
+    """
+
+    encoder = wpilib.AnalogPotentiometer
+    motor = wpilib.Talon
+    stop_value = 0
+    extended = False
+
+    def set_stop_value(self, stop_value):
+        self.stop_value = stop_value
+
+    def set_extended(self, extended):
+        self.extended = extended
+
+    def execute(self):
```

Comment:
How about some tests to make sure the logic we intend actually works? Perhaps you can work with someone new to Python programming to do this work?

Reply:
That would be a great idea, but all that logic is going to be replaced with PID control. However, it's still a good idea to teach some unit testing using that piece of logic.